### DISCUSSION TITLE
the package "zendframework/zend-db": "dev-directus" is not available

### DISCUSSION THREAD

**Discussion by shmc** [directus/api#2361] [Q&A] [answered]
After all the changes on version 8 the package "zendframework/zend-db": "dev-directus" is not available. can you make it available so we can continue to build it?
Are there any changes between the zendframework:zend-db:2.11.0 version and one used on Directus?

**Comment by rijkvanzanten** [comment]
Yes, lets see how to expose that

**Comment by rijkvanzanten** [comment, marked as answer]
Updated the composer.json file to pull from the local packages, and added a `build.sh` script you can use to bundle the archive to a release build 👍
